core/src/main/java/org/bitcoinj/params/TestNet3Params.java

```diff
@@ -163,123 +163,164 @@ public String getPaymentProtocolId() {
+        "35.162.18.116",
+        "35.91.208.56",
+        "34.219.33.231",
+        "52.34.250.214",
+        "35.91.134.89",
+        "50.112.58.114",
+        "54.191.146.137",
+        "34.218.66.37",
+        "34.221.196.103",
+        "35.91.157.30",
+        "34.221.102.51",
+        "18.237.165.242",
+        "52.37.61.9",
+        "54.212.89.127",
+        "34.209.238.228",
+        "35.92.143.7",
+        "35.89.113.195",
+        "52.12.54.89",
+        "34.219.153.30",
+        "34.215.171.237",
+        "54.70.243.3",
+        "54.184.126.25",
+        "34.222.85.18",
+        "34.221.252.179",
+        "35.85.33.152",
+        "54.200.220.105",
+        "54.245.75.47",
+        "54.214.59.174",
+        "35.164.77.177",
+        "35.89.66.84",
+        "35.91.150.34",
+        "35.92.219.124",
+        "34.222.82.127",
+        "34.220.171.156",
+        "35.90.42.64",
+        "35.89.53.128",
+        "35.93.151.188",
+        "34.211.172.212",
+        "34.220.118.79",
+        "34.220.187.233",
+        "34.220.85.81",
+        "35.167.165.224",
+        "34.210.26.93",
+        "35.90.53.180",
+        "34.214.48.68",
+        "35.166.18.166",
+        "35.165.50.126",
+        "52.42.202.128",
+        "52.12.176.90",
+        "44.233.44.95",
+        "35.167.145.149",
+        "52.34.144.50",
+        "44.240.98.102",
+        "54.201.32.131",
+        "52.10.229.11",
+        "52.13.132.146",
+        "44.228.242.181",
+        "35.82.197.197",
+        "52.40.219.41",
+        "44.239.39.153",
+        "54.149.33.167",
+        "35.164.23.245",
+        "52.33.28.47",
+        "52.43.86.231",
+        "52.43.13.92",
+        "35.163.144.230",
+        "52.89.154.48",
+        "52.24.124.162",
+        "44.227.137.77",
+        "35.85.21.179",
+        "54.187.14.232",
+        "54.68.235.201",
+        "52.13.250.182",
+        "35.82.49.196",
+        "44.232.196.6",
+        "54.189.164.39",
+        "54.213.204.85",
+    };
+
+    public static String[] HP_MASTERNODES = {
+        "34.214.48.68",
+        "35.166.18.166",
+        "35.165.50.126",
+        "52.42.202.128",
+        "52.12.176.90",
+        "44.233.44.95",
+        "35.167.145.149",
+        "52.34.144.50",
+        "44.240.98.102",
+        "54.201.32.131",
+        "52.10.229.11",
+        "52.13.132.146",
+        "44.228.242.181",
+        "35.82.197.197",
+        "52.40.219.41",
+        "44.239.39.153",
+        "54.149.33.167",
+        "35.164.23.245",
+        "52.33.28.47",
+        "52.43.86.231",
+        "52.43.13.92",
+        "35.163.144.230",
+        "52.89.154.48",
+        "52.24.124.162",
+        "44.227.137.77",
+        "35.85.21.179",
+        "54.187.14.232",
+        "54.68.235.201",
+        "52.13.250.182",
+        "35.82.49.196",
+        "44.232.196.6",
+        "54.189.164.39",
+        "54.213.204.85",
     };
 
     @Override
     public String[] getDefaultMasternodeList() {
         return MASTERNODES;
```

Comment:
Shouldn't this return a combined list of `MASTERNODES` + `HP_MASTERNODES`?  I assumed hp-masternodes can still be used for all the regular functions.